packages/base/adapters/solana/web3js/utils/defaultSolanaConfig.ts

```diff
@@ -3,18 +3,13 @@ import '@web3modal/polyfills'
 import type { Chain, Metadata, Provider, ProviderType } from './scaffold/index.js'
 
 declare global {
-  interface Navigator {
```

Comment:
Why is this removed? Make sure it's aligned with `main` branch. Shouldn't be responsibility of this PR